src/main/resources/templates/templateDslBuilder.ftlh

```diff
@@ -105,6 +107,10 @@
             pactDslJsonBody.${field.functionByType}("${field.name}", (int) ${field.name});
           <#elseif field.fieldType == "BigInteger">
             pactDslJsonBody.${field.functionByType}("${field.name}", ${field.name}.intValue());
+          <#elseif field.fieldType == "float">
+            pactDslJsonBody.${field.functionByType}("${field.name}", (double) ${field.name});
+          <#elseif field.fieldType == "byte">
+            pactDslJsonBody.${field.functionByType}("${field.name}", (int) ${field.name});
```

Comment:
I can see the Long type is missing.

Reply:
pactDslJsonBody.integerType suports ints and longs
that is why I cast short and byte to int

pactDslJsonBody.decimal supports floating point and decimal
that is why I cast float to double